Core/Rewriters/MockSetupRewriter.cs

```diff
@@ -293,7 +307,7 @@ private static string GetTimesValue (ExpressionStatementSyntax originalNode)
           1 when invocationExpression.ArgumentList.Arguments.Single().Expression is LiteralExpressionSyntax { Token: { Value: int times } } => times.ToString(),
           2 when invocationExpression.ArgumentList.Arguments.First().Expression is LiteralExpressionSyntax { Token: { Value: int min } }
                  && invocationExpression.ArgumentList.Arguments.Last().Expression is LiteralExpressionSyntax { Token: { Value: int max } } => $"{min}:{max}",
-          _ => throw new InvalidOperationException ("Unable to resolve Repeat.Times")
+          _ => "-42"
```

Comment:
🤔

Reply:
Q: Why -42?
A: This is supposed to make the test fail, because it cannot be repeated -42 times.

Q: When does this happen?
A: This problem occurred in some dp projects, where some tests had more complex `Repeat.Times` expressions, e.g. `Repeat.Times (anyValue - 1)`